more_autograding_examples/docker_choice_of_language/config/config.json

```diff
@@ -0,0 +1,72 @@
+{
+    "required_capabilities": "default",
+    "autograding_method": "docker",
+    "assignment_message": "Please submit your code in one of 3 languages. Only submit to a single box",
+    "part_names": ["Rust", "C", "C++"],
+
+    "one_part_only": true,
+
+    "autograding": {
+        "submission_to_compilation": ["**/*.rs", "**/*.c", "**/*.h", "**/*.cpp", "**/*.hpp"]
+    },
+    "testcases": [
+        {
+            "type": "Compilation",
+            "title": "Rust Compilation",
+            "containers": [
+                {
+                    "commands": "/usr/local/cargo/bin/rustc part1/*.rs -o a.out -C target-feature=+crt-static",
```

Comment:
I thought all rust binaries were static. What problem did you encounter leading you to add `-C target-feature=+crt-static`? And could you throw a comment here explaining it too for future reference.

Reply:
I found that rust was using dynamic linking by default and that the libraries were not available in the default autograding image.